.travis.yml

```diff
@@ -0,0 +1,61 @@
+sudo: false
+dist: trusty
+
+language: php
+
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+matrix:
+  include:
+    - php: 7.2
+      env: WP_VERSION=latest
+    - php: 7.1
+      env: WP_VERSION=latest
+    - php: 7.0
+      env: WP_VERSION=latest
+    - php: 7.1
+      env: WP_VERSION=trunk
+    - php: 7.1
+      env: WP_VERSION=trunk
+    - php: 7.0
+      env: WP_TRAVISCI=phpcs
+  fast_finish: true
+  allow_failures:
+    - php: 7.2
```

Comment:
Now that VIP is moving to 7.2, do we want to allow failures on 7.2?

Reply:
Good call. I don't think so, so I'll remove this.